deltacat/aws/constants.py

```diff
@@ -3,6 +3,10 @@
 from deltacat.utils.common import env_integer, env_string
 
 DAFT_MAX_S3_CONNECTIONS_PER_FILE = env_integer("DAFT_MAX_S3_CONNECTIONS_PER_FILE", 8)
-BOTO_MAX_RETRIES = env_integer("BOTO_MAX_RETRIES", 15)
+BOTO_MAX_RETRIES = env_integer("BOTO_MAX_RETRIES", 5)
```

Comment:
Any reason for lowering the defaults if they are adjustable through environment values?

Reply:
This was discussed during daft offsite. We anyway have tenacity retries.